src/common/Icons.js

```diff
@@ -108,3 +108,4 @@ export const IconGroup: SpecificIconType = makeIcon(FontAwesome, 'group');
 export const IconPlus: SpecificIconType = makeIcon(Feather, 'plus');
 // eslint-disable-next-line react/function-component-definition
 export const IconWebPublic: SpecificIconType = props => <ZulipIcon name="globe" {...props} />;
+export const IconAllMessages: SpecificIconType = props => makeIcon(FontAwesome, 'align-left');
```

Comment:
Take a look again at how this line compares to the existing examples in the file. I think this is the reason why this version doesn't work, showing a blank icon.